tests/loaders.test.ts

```diff
@@ -35,7 +35,7 @@ Deno.test("load the pages of a site", async () => {
     // @ts-ignore: unknown property
     equals(page.data.colors.length, 3);
     // @ts-ignore: unknown property
-    equals(page.data.documents.length, 3);
+    equals(page.data.documents.content.length, 3);
```

Comment:
I don't know how this test broke, apparently `documents` was the array before and now it has `.dependencies` and `.content`. I can see how adding dependencies affected this but I couldn't find the code that injected the array in a different place now.

I'm not confident about this and help is much appreciated.